src/cdk/utils/basic-authorizer-handler/index.ts

```diff
@@ -39,10 +39,27 @@ function createAllowPolicy(
   };
 }
 
+export function getAuthHeaderValue(headers?: {
```

Comment:
You can move this function to its own file `src/cdk/utils/get-auth-header-value.ts` then 100% coverage should be achievable.

Reply:
I think you could also rewrite this function to be generic. Simply pass the headerName as second argument. And name the function `getHeaderValue`

Reply:
I just see outsourcing is not an option. We're not bundling the file. So the generic one doesn't make sense anymore either. Well then you can exclude the lambda in the Jest Config for coverage.

Reply:
Splitted into `getHeaderValue()` and `getAuthHeaderValue()` within the same file. The latter function exists only to avoid defining constants for header names / avoid typos when used somewhere else.